scripts/mempool.js

```diff
@@ -96,27 +99,32 @@ class Mempool {
         return false;
     }
 
-        /**
+    /**
      * Check if an exact UTXO match can be found in our wallet
      * @param {id,path,vout} - txid path and vout of the UTXO
      * @returns {Boolean} `true` or `false`
      */
-         isAlreadyStored({id,path,vout,status}) {
-            for (const cUTXO of this.UTXOs) {
-                if (cUTXO.id === id && cUTXO.path===path && cUTXO.vout===vout && cUTXO.status===status) {
-                    return true;
-                }
+    isAlreadyStored({ id, path, vout, status }) {
```

Comment:
With the es-lint rules this overwrites the prior declared isAlreadyStored(cNewUTXO)

We will have to rename the function and appropriate calls.

Reply:
Yes, this will be fixed in #65